openprescribing/media/js/bar-charts.js

```diff
@@ -127,3 +127,7 @@ var barChart = {
 };
 
 barChart.setUp();
+
+$(document).ready(function() {
```

Comment:
Might be worth checking that the global `doorbell` variable exists, because I don't think the DOM being ready will guarantee that.

Also, I've just observed that the rest of the code isn't inside `document.ready`, which it maybe should be...